src/axios-har-tracker.ts

```diff
@@ -96,17 +168,42 @@ async function axiosTracker(call) {
             _blocked_queueing: -1
           }
         };
-        const enteriesContent = Object.assign({}, newEntry);
-        generatedHar.log.entries.push(enteriesContent);
+        this.entriesContent = Object.assign({}, this.newEntry);
+        this.generatedHar.log.entries.push(this.entriesContent);
         return resp;
       }
-    },
-    error => {
-      return Promise.reject(error);
+    );
+  }
+
+  private errorHandler(error){
```

Comment:
this function is just one line, same as Promise.reject(error), I think there is no need to create a separate method for it.